sdk/storage/azure-storage-files-datalake/src/datalake_file_client.cpp

```diff
@@ -323,7 +323,12 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Models::ReadFileResult ret;
     ret.Body = std::move(result->BodyStream);
     ret.HttpHeaders = FromBlobHttpHeaders(std::move(result->HttpHeaders));
-    ret.ContentRange = std::move(result->ContentRange);
+    // FIXME
```

Comment:
I appreciate you adding this test case, but to decouple the work, it would be best to comment it out in your PR. I'll fix it in follow up PRs.